# map_filename: /home/john/gestelt_ws/src/gestelt/dynamic_brushfire/maps/empty_room.pgm
# map_filename: /home/john/gestelt_ws/src/gestelt/dynamic_brushfire/maps/map_circular.pgm
map_filename: /home/john/gestelt_ws/src/gestelt/dynamic_brushfire/maps/map.pgm
resolution: 0.1
negate: false
occ_threshold: 0.65
free_threshold: 0.196
origin_x: 0.0
origin_y: 0.0
origin_z: 0.0
